util.go

```diff
@@ -51,3 +53,13 @@ func BashComplete(c *cli.Context) {
 		PrintFlagNames(c.App.VisibleFlags(), c.App.Writer)
 	}
 }
+
+func ConfigPath() (string, error) {
```

Comment:
Can you please add doc string for this function? It is not obvious what configuration file is actually read?

Reply:
sure thing